### ISSUE TITLE
Audit page widths on registrar and get.gov

### ISSUE THREAD

**Issue by Katherine-Osos** [cisagov/manage.get.gov#1628]
### Issue description

We have a few different page widths used on get.gov and the registrar. For example, within the domain-request form, there are two widths: one for "instruction" content and one of "form" content. Here's an example that highlights the two widths: 

![Image](https://github.com/cisagov/manage.get.gov/assets/119689946/7e09c80f-a96a-42e1-b19d-e1cdc9a019d4)

 

### Acceptance criteria

- [ ] Audit the registrar and determine whether any adjustments need to be made to our current widths
- [ ] Audit get.gov and determine whether any adjustments need to be made to our current widths
- [ ] Create a dev ticket to implement any changes to the widths

**Comment by katypies** [comment]
I think this is still an issue and worth assessing/resolving. Putting in design parking lot to evaluate, especially as many of our pages have changed, so this may be a bigger or smaller lift/issue now!

**Comment by witha-c** [comment]
@gabo0oo has this issue been addressed in other tickets?

**Comment by gabo0oo** [comment]
@witha-c No, this is still a thing because it's a setting that the devs have at a very core style level (may even be a USWDS thing) so it needs to be over written.

**Comment by h-m-f-t** [comment]
Prioritizing this in favor of #4806, which may have some useful notes.

@gabo0oo and @katypies, can you spend a few minutes to refine this a bit more and help us know when to prioritize?

**Comment by katypies** [comment]
I pushed a change to the product sandbox yesterday for design to review and us to walk through. We may end up not doing this audit if the changes can be made quickly and resolve most of the concerns for now.

**Comment by katypies** [comment]
I had hoped that a quick change or two might resolve the immediate visual discrepancies, and while it _might_ in some ways, it's going to potentially create others. We have sone inconsistent application of when we start the form within the request flow (for example, right after the first heading, versus after the initial instructions). Some of the visual sense that things _aren't quite right_ also stems from some repetitive information that we're putting in the fieldsets, form field titles, and legend-type information. In the example here, we're actually asking the same question 3-4 times about "what tribe are you associated with" which can be skimmed visually, or skipped, but might pose some concerns from an accessibility point of view.

All of this is to say that we should do an audit, and I think it might be something where we want to start with a template of how we approach this, and then make sure that the forms and form steps all conform to that template.

I'm leaving this open, but going to parking lot it to come back and map out next steps for tickets.
